src/sidebar/components/NotebookView.js

```diff
@@ -129,14 +129,16 @@ function NotebookView({ loadAnnotationsService, streamer }) {
   }`;
 
   return (
-    <div className="NotebookView">
-      <header className="NotebookView__heading" ref={threadListScrollTop}>
-        <h1 className="NotebookView__group-name">{groupName}</h1>
+    <div className="grid gap-2 lg:grid-cols-2" data-testid="notebook-container">
+      <header className="leading-none lg:col-span-2" ref={threadListScrollTop}>
+        <h1 className="text-2xl font-bold" data-testid="notebook-group-name">
+          {groupName}
+        </h1>
       </header>
-      <div className="NotebookView__filters">
+      <div>
```

Comment:
Does this `<div>` serve a purpose now that it is unstyled?

Reply:
It's an explicit grid-item (immediate child of grid). It _might_ not be necessary, but that would be trusting that the rendered element of `NotebookFilters` is a block element. Strictly not needed, but not harmful either.

Reply:
Ah, makes sense. I suppose if `NotebookFilters` changed from rendering one to zero or multiple items that could have unexpected consequences otherwise.